daffodil-core/src/main/scala/org/apache/daffodil/runtime1/ElementBaseRuntime1Mixin.scala

```diff
@@ -81,6 +81,9 @@ trait ElementBaseRuntime1Mixin { self: ElementBase =>
 
     // For simple elements with text representation, valueLength is captured in
     // individual parsers since they handle removing delimiters and padding.
+    // Related, simple types that aren't text (e.g. hex binary, packed decimal)
+    // but are delimited also use the same test delimited parsers that capture
```

Comment:
"the same test delimited parsers" ???

Reply:
should be "text delimited parsers", will fix